examples/headless/src/main.rs

```diff
@@ -145,7 +146,7 @@ async fn render(mut scenes: SceneSet, index: usize, args: &Args) -> Result<()> {
             .args
             .base_color
             .or(scene_params.base_color)
-            .unwrap_or(vello::peniko::Color::BLACK),
+            .unwrap_or(palette::css::BLACK),
```

Comment:
I wonder if it would actually be more idiomatic to import the full `css` palette, i.e. this would be `css::BLACK`.

The `palette` identifier here reads a bit out of place.

Reply:
I've gone back and forth with this a couple of times in my changes. This indicates that it is coming from a palette of colors and not that it has anything to do directly with CSS.

Reply:
If we aren't careful, I'll end up with `use peniko::color::palette::css as css_palette;` !

Reply:
I read `css::BLACK` as `CSS black`, i.e. black"from CSS", so as defined in CSS. It seems like Bevy mostly always imports them fully, so would use `BLACK` here.

(The swapped capitalisation in that first sentence is quite funny)

Reply:
This doesn't need to be resolved now, and so can land as-is.